### DISCUSSION TITLE
Add password on interface or single room planned?

### DISCUSSION THREAD

**Discussion by Dayk0** [mozilla/hubs#4301] [Hubs Cloud Questions] [answered]
Hello,

I would like to know if a password system for limited access to an application is planned in the coming months or not?
At the moment everyone can access the public rooms and I would like to limit it to a certain number of people.
Or a password at the entrance to the room which would limit this access.

Thank you.

**Comment by Softvision-GeluHaiduc** [comment, marked as answer]
Hello @Dayk0 

For the first question if you wish you can log a feature request detailing the feature and describing the solution you'd like. (this does not guarantee the feature will be implemented but it's a start) 
For the second question in the room settings under the Room Access settings, you could change the setting to “Invite only” which will generate a room link with an additional unique identifier and this invite link can be also revoked. For more information, you can take a look at the hubs documentation room-permissions section.

Marking this as answered, unmark the answer if you need any more information. Thank you!
